docs/kubernetes/how-to-migrate-from-k8s-alpha-to-terraform/index.md

```diff
@@ -0,0 +1,257 @@
+
+        kubectl --kubeconfig=mycluster.conf get nodes
+
+1.  The output will then list an entry for each node:
+
+    {{< output >}}
+kubectl --kubeconfig=mycluster.conf get nodes
+NAME             	STATUS   ROLES	  AGE 	  VERSION
+mycluster-master-1      Ready	 master   21m 	  v1.13.6
+mycluster-node-1 	Ready	 &lt;none&gt;   18m 	  v1.13.6
+mycluster-node-2 	Ready	 &lt;none&gt;   18m 	  v1.13.6
+mycluster-node-3 	Ready	 &lt;none&gt;   18m 	  v1.13.6
+mycluster-node-4 	Ready	 &lt;none&gt;   4m26s   v1.13.6
+mycluster-node-5 	Ready	 &lt;none&gt;   4m52s   v1.13.6
+{{< /output >}}
+
+### Upgrade a Cluster
+
+You may have noticed that the Terraform module, `cluster.tf`, refers to a specific branch or git commit hash referencing a remote Terraform module on GitHub. The following section will outline how to upgrade your cluster to the latest version of this module.
+
+For example, your `source` variable may have a value that points to the git branch ref `for-cli`. To perform an upgrade this must point to the latest commit history hash.
+
+1.  Visit the branch of the Terraform module you're using on [Github](https://github.com/linode/terraform-linode-k8s/commits/for-cli). Note the commit history and copy the latest hash by clicking on the clipboard next to the hash of the most recent commit. At the time of this writing, the most recent hash is as follows:
+
+    {{< output >}}
+5e68ff7beee9c36aa4a4f5599f3973f753b1cd9e
+{{< /output >}}
+
+1.  Edit `cluster.tf` to prepare for the upgrade. Update the following section using the hash you copied to appear as follows:
+
+    {{< file "cluster.tf" >}}
+source = "git::https://github.com/linode/terraform-linode-k8s.git?ref=5e68ff7beee9c36aa4a4f5599f3973f753b1cd9e"
+{{< /file >}}
+
+1.  To apply these changes, re-init the modules by running the following command:
+
+        terraform init
+
+1.  Once this has completed, run terraform apply:
+
+        terraform apply
+
+    {{< note >}}
+Depending on the changes that have been configured, you may or may not see the upgrade perform actions. For example, in this case, because the only change was to the kubernetes version, no actions were taken.
+{{< /note >}}
+
+### Delete a Cluster
+
+To destroy a cluster, navigate to the directory containing your cluster's files, and enter the `terraform destroy` command:
+
+    cd ~/.k8s-alpha-linode/mycluster
+    terraform destroy
+
+Terraform will prompt you to confirm the action, and on confirmation will proceed to destroy all associated resources. If this process is interrupted for any reason, you can run the command again at any time to complete the process.
+
+## Create a Cluster
+
+1.  Create a new directory to house its configuration files in the `~/.k8s-alpha-linode` directory. In this example the cluster name is `mynewcluster`:
```

Comment:
to house the new cluster's configuration files

Reply:
updated